tests/target/issue-5358.rs

```diff
@@ -0,0 +1,4 @@
+// Test /* comment */ inside trait generics gets duplicated.
```

Comment:
Should this read `Test /* comment */ inside trait generics does not get duplicated`?

Reply:
Yes! I'll fix it un a few minutes.

Reply:
Done!